### DISCUSSION TITLE
Sensor node traffic not displaying under Manager node

### DISCUSSION THREAD

**Discussion by arjoshi89** [Security-Onion-Solutions/securityonion#8187] [Unsupported Versions]
Distributed: Master node and 2 heavy nodes (Sensor 1 and Sensor 2)
Running 2.3.90

Grid status showing green for both sensor 1 and sensor 2

**Sensor2 not showing under Hunt interface of UI for current month timeline ?**
Only Sensor1 showing

From master node:

[root@securityonion ~]#  salt '*' test.ping
sensor1_heavynode:
    True
securityonion_manager:
    True
sensor2_heavynode:
    True

Disk usage high under sensor 2
[root@sensor2 /]# df -kh
Filesystem               Size  Used Avail Use% Mounted on
devtmpfs                 5.8G     0  5.8G   0% /dev
tmpfs                    5.8G  120K  5.8G   1% /dev/shm
tmpfs                    5.8G   11M  5.8G   1% /run
tmpfs                    5.8G     0  5.8G   0% /sys/fs/cgroup
**/dev/mapper/system-root   91G   91G   53M 100% /**
/dev/sda2                497M  197M  300M  40% /boot
/dev/mapper/system-nsm   179G  159G   20G  89% /nsm
/dev/mapper/system-tmp   2.0G   34M  2.0G   2% /tmp
overlay                   91G   91G   53M 100% /var/lib/docker/overlay2/1781a60b13acad7e3c3d8e4f233605dcf3cb69274443582b6db124407fb3f5b6/merged
overlay                   91G   91G   53M 100% /var/lib/docker/overlay2/1f3e8f74ca182a86b0f7b3a000f11d66206e0891f987c7f3727e6fb4069d9374/merged
overlay                   91G   91G   53M 100% /var/lib/docker/overlay2/3da3f78d820977a35f789c2a8a6d7dc5439cde3cc65452ba0aeba68f49d27b40/merged
overlay                   91G   91G   53M 100% /var/lib/docker/overlay2/46e02b11b807e9e391f141d52922473a6b11e8ff24df159a1b6946bb2610719e/merged
overlay                   91G   91G   53M 100% /var/lib/docker/overlay2/cf4d9722dfbf3ca04a392397cc65c0c1e48d6a3a3c3b22e5859649983fa45a6c/merged
overlay                   91G   91G   53M 100% /var/lib/docker/overlay2/b01e712be755462922770ea189773a8cf9531b471f40488e8ba4dfa3e3386c64/merged
overlay                   91G   91G   53M 100% /var/lib/docker/overlay2/1b4177aa7e37b254cd3fb4aa9fe8f0d22b73eb634a273d669b255d2864c87c84/merged
overlay                   91G   91G   53M 100% /var/lib/docker/overlay2/c7d8931c4d6baa45f1dac1e51018c4f543f58e9100dadf44edc817b4f7c8a9dd/merged
overlay                   91G   91G   53M 100% /var/lib/docker/overlay2/8f5c964793147783691559191d9b25943403003865923f053c5dfedbb2af942a/merged
overlay                   91G   91G   53M 100% /var/lib/docker/overlay2/c534f872a1f3baf155529f1127b140d76b5791a673bbddf865eb3983031f5922/merged
overlay                   91G   91G   53M 100% /var/lib/docker/overlay2/0d947967e4df606099ba4271b941b8c077880457ea7918fcef0512d23c288da8/merged
overlay                   91G   91G   53M 100% /var/lib/docker/overlay2/582adfef1c6f40b400104fb87ab5255c916b94ea30893a235c3f29f1eb5a34e7/merged
overlay                   91G   91G   53M 100% /var/lib/docker/overlay2/14b0bdaa97efedb942e17048e50bc19c733b58a8302c673a6589ef1bf6f1dc30/merged
overlay                   91G   91G   53M 100% /var/lib/docker/overlay2/f94e21b43dd6ec5b75c4a9733703ef0e3a1e813bb68d9039af1b5a2f3bcd052f/merged
overlay                   91G   91G   53M 100% /var/lib/docker/overlay2/7d0c74c114a51e76f997d9c82a273bc8b1d79a270426f87947c2b9365706c735/merged
overlay                   91G   91G   53M 100% /var/lib/docker/overlay2/62eedc1ef9b5d965d028457ffacfd273f6913f6d44d6634c3cbdee1b87b5cd6f/merged
tmpfs                    1.2G     0  1.2G   0% /run/user/1000
overlay                   91G   91G   53M 100% /var/lib/docker/overlay2/fc1df7a7bea4e3afa576ab201187969437223030d6713946db0cefd6403dc84f/merged
[root@sensor2 /]#

[root@sensor2 /]# so-status

Checking Docker status

    Docker --------------------------------------------------------------------------------------------------------------------------------------- [ OK ]

Checking container statuses

    so-curator ----------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-elasticsearch ----------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-filebeat ---------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-logstash ---------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-nginx ------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-redis ------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-sensoroni --------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-steno ------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-strelka-backend --------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-strelka-coordinator ----------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-strelka-filestream ------------------------------------------------------------------------------------------------------------------------ [ OK ]
    so-strelka-frontend -------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-strelka-gatekeeper ------------------------------------------------------------------------------------------------------------------------ [ OK ]
    so-strelka-manager --------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-suricata ---------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-telegraf ---------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-zeek -------------------------------------------------------------------------------------------------------------------------------------- [ OK ]

[root@sensor2 /]#

**Comment by weslambert** [comment]
I would first address this (the lack of disk space):

`Disk usage high under sensor 2`
...
`/dev/mapper/system-root 91G 91G 53M 100% /`

After addressing the disk space issue, ensure the services are restarted or that the machine is rebooted.

If you continue to have issues after clearing disk space and restarting services/rebooting, I would try having a look at the logs in `/opt/so/log/elasticsearch` for `sensor2`.

**Reply by arjoshi89** [reply]
I noticed this space decreased to 94% today but still the same issue under UI:
/dev/mapper/system-root   91G   85G  6.0G  94% /

How can I find out what is taking more space?

sensor2.log is filled with below lines:
[2022-06-27T13:40:17,934][DEPRECATION][org.elasticsearch.deprecation.cluster.metadata.IndexNameExpressionResolver] this request accesses system indices: [.security-7], but in a future major version, direct access to system indices will be prevented by default
[2022-06-27T13:40:17,934][DEPRECATION][org.elasticsearch.deprecation.cluster.metadata.IndexNameExpressionResolver] this request accesses system indices: [.security-7], but in a future major version, direct access to system indices will be prevented by default
[2022-06-27T13:40:18,861][DEPRECATION][org.elasticsearch.deprecation.cluster.metadata.IndexNameExpressionResolver] this request accesses system indices: [.security-7], but in a future major version, direct access to system indices will be prevented by default

**Reply by dougburks** [reply]
You can run a command like this to see how much space each of the top-level directories are using:
```
sudo du --max-depth=1 -h /
```

However, it looks like your disk space is only slightly above the bare minimum requirements:
https://docs.securityonion.net/en/2.3/hardware.html#minimum-specs

You probably want much more disk space for a production deployment.
